internals/onepassword/cli_v2.go

```diff
@@ -0,0 +1,137 @@
+package onepassword
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type OPV2CLI struct {
+	version string
```

Comment:
Same comment as above:
" Do we actually ever use the version, or are we just passing it around? If not, I think making the struct, `OPV1CLI` or `OPV2CLI`, provides enough information, and we should get rid of the `version` attribute. "